test/WebSites/ControllersFromServicesWebSite/Views/Another/InServicesTagHelper.cshtml

```diff
@@ -0,0 +1,2 @@
+@addTagHelper *, ControllersFromServicesWebSite
+<inservices></inservices>
```

Comment:
Would be really good to see this use one of the built-in taghelpers since that requires specific handling